gamepad-host.js

```diff
@@ -218,4 +228,23 @@
         sendData('gamepad', state);
     }
 
+    function launchIntoFullscreen(element) {
+      if(element.requestFullscreen) {
+        element.requestFullscreen();
+      } else if(element.mozRequestFullScreen) {
+        element.mozRequestFullScreen();
+      } else if(element.webkitRequestFullscreen) {
+        element.webkitRequestFullscreen();
+      } else if(element.msRequestFullscreen) {
+        element.msRequestFullscreen();
+      }
+    }
```

Comment:
it'd be cool to also lock orientation à la

https://github.com/mozilla/galaxy.js-mobile-gamepad/blob/master/src/js/lib/utils.js#L138-L153